config/sync/core.entity_view_display.node.gp_practice.default.yml

```diff
@@ -16,6 +16,7 @@ dependencies:
     - field.field.node.gp_practice.field_gp_surgery_name
     - field.field.node.gp_practice.field_location
     - field.field.node.gp_practice.field_meta_tags
+    - field.field.node.gp_practice.field_phone
     - field.field.node.gp_practice.field_telephone
```

Comment:
field_telephone is still listed here as a dependency, although it's replaced by field_phone across the board. Can we take this out of the list of config dependencies?